### PR TITLE
[WC-1357]: rename HTML Node to HTML Element

### PR DESCRIPTION
**Pull request by r0b1n**
#### Feature specific

-   Comply with designs ✅ 
-   Comply with PM's requirements ✅ 

## This PR contains

-   [ ] Bug fix
-   [x] Feature
-   [ ] Refactor
-   [ ] Documentation
-   [ ] Other (describe)

## What is the purpose of this PR?

HTML Element is a better name for the widget.

### CODE REVIEW

File: packages/pluggableWidgets/html-element-web/src/HTMLElement.editorConfig.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const disabledElements = ["script"];
 
 function isValidHtmlTagName(name: string): boolean {
     // rather naive approach that works in most cases
-    return /^[a-z_][\w.-]*$/i.test(name);
+    return /^[a-z][\w.-]*$/i.test(name);
```
Comment (iobuhov) [inline packages/pluggableWidgets/html-element-web/src/HTMLElement.editorConfig.ts:28]: @r0b1n , are you sure we allow `-`?

**Review by iobuhov — COMMENTED** [review]
Are you sure `html-element-root` is fine class for widget?
AFAIK we trying to use `widget-<widget-name>` pattern

File: packages/pluggableWidgets/html-element-web/src/HTMLElement.editorConfig.ts (hunk 1 above)
Comment (r0b1n) [reply packages/pluggableWidgets/html-element-web/src/HTMLElement.editorConfig.ts:28]: Yes, while no native html tags use `-`, we want to support web components and they are named with 2 or more words separated with `-` (most of the time at least).

**Comment by r0b1n** [comment]
`html-element-root` is only in test files/snapshots. But good that you pointed this out. The actual class name was `html-element-widget` which is still off. I changed it to `widget-html-element`. Thanks!

File: packages/pluggableWidgets/html-element-web/src/HTMLElement.editorConfig.ts (hunk 1 above)
Comment (iobuhov) [reply packages/pluggableWidgets/html-element-web/src/HTMLElement.editorConfig.ts:28]: Ah, okay, I got it

**Review by iobuhov — APPROVED** [review]

**Pull request merged**